app/views/products/show.html.erb

```diff
@@ -0,0 +1,45 @@
+
+<div class="row">
+  <% if flash[:error] %>
+    <div class="alert alert-danger" role="alert">
+      <%= flash[:error] %>
+    </div>
+  <% end %>
+
+<div class="col-md-4 col-center">
+  <p><%= image_tag(@product.photo_url, size: "300x300") %></p>
+</div>
+
+<div class="col-md-8" >
+<h4><%= @product.name %></h4>
+<p>Price: $<%= @product.price %></p>
+<p>Description: <%= @product.description %></p>
+<p>Stock: <%= @product.stock %></p>
+<p>Merchant: <%= @product.user.username %></p>
+<p>Categories:
+<% @product.category.all.each do |category| %>
+  <%= link_to "#{category.name}", category_path(category)  %>
+<% end %>
+
+<% unless @reviews == nil %>
```

Comment:
It seems like it might be worthwhile to show a "No Reviews Yet" section or something like that if there haven't been any reviews yet